setup.py

```diff
@@ -41,4 +41,5 @@
     ],
     packages=find_packages(exclude=["cloudevents.tests"]),
     version=pypi_config["version_target"],
+    install_requires=["deprecation==2.1.0"],
```

Comment:
```suggestion
    install_requires=["deprecation>=2.0,<3.0"],
```
We shouldn't pin sub-dependencies of libraries.